packages/venia-concept/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "build": "yarn run build:prod",
     "build:analyze": "yarn run clean && mkdir dist && webpack -p --profile --no-progress --env.mode production --json > dist/build-stats.json && webpack-bundle-analyzer dist/build-stats.json",
-    "build:debug": "node --inspect-brk ./node_modules/.bin/webpack --no-progress --env.mode",
+    "build:debug": "node --inspect-brk ./node_modules/.bin/webpack --no-progress --env.mode development",
```

Comment:
Any reason this went from `none` to `development`?

Reply:
This was broken long ago. We never tried the script so we never found it. I had to use this script to debug the code and that's how I found the issue.